Microsoft.Toolkit.Uwp.UI.Controls/Menu/MenuItem.cs

```diff
@@ -309,6 +320,24 @@ private void MenuFlyout_Opened(object sender, object e)
             IsOpened = true;
             VisualStateManager.GoToState(this, "Opened", true);
             _parentMenu.IsInTransitionState = false;
+
+            if (!_menuFlyoutRepositioned)
```

Comment:
Is opened called multiple times in a row? Why check when for this variable when it's set to false on close?